### PR TITLE
Fix feed tip tile and other tipping QA issues

### PR DESCRIPTION
**Pull request by sddioulde**
### Description

Fix feed tip tile and other tipping QA issues

what was fixed?
- Missing padding on supporting box in artist popover when full width
- Show top supporters list modal on clicking "was tipped by" section of feed tip tile
- Separate artist chip touch targets
- Fix spacing in user lists
- Align user list photo to name at top of row
- Feed tip tile badge size
- Dismiss button size
- Add a skeleton tile while we’re loading recent tips
- Hovering on feed tip tile and it will underline and be clickable
- Showing popover for feed tip tile name

### Dragons

n/a

### How Has This Been Tested?

local agains stage

### How will this change be monitored?

n/a

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-tip-qa

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-tip-qa

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-tip-qa

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-tip-qa

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-tip-qa

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-tip-qa

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-tip-qa

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-tip-qa

File: packages/web/src/components/tipping/feed-tip-tile/FeedTipTile.tsx [hunk 1]
```diff
@@ -36,32 +58,51 @@ const WasTippedBy = () => (
...
+        id: receiver.user_id
+      })
+    )
+    dispatch(setVisibility(true))
+  }, [dispatch, receiver])
+
+  return (
+    <div className={styles.tippers} onClick={handleClick}>
+      {tippers.slice(0, NUM_FEED_TIPPERS_DISPLAYED).map((tipper, index) => (
+        <div key={`tipper-${tipper.user_id}`} className={styles.tipperName}>
```
[the remark this answers is not in the document]
Comment (sddioulde) [reply packages/web/src/components/tipping/feed-tip-tile/FeedTipTile.tsx:81]: slightly afraid that the user id is used somewhere else on the page

File: packages/web/src/components/tipping/feed-tip-tile/FeedTipTile.tsx [hunk 2]
```diff
@@ -36,32 +58,51 @@ const WasTippedBy = () => (
...
+          <span>{tipper.name}</span>
+          <UserBadges
+            userId={tipper.user_id}
+            className={styles.badge}
+            badgeSize={12}
+            inline
+          />
+          {index < tippers.length - 1 &&
```
[the remark this answers is not in the document]
Comment (sddioulde) [reply packages/web/src/components/tipping/feed-tip-tile/FeedTipTile.tsx:89]: yup. we're slicing NUM_FEED_TIPPERS_DISPLAYED, so imagine that number is really big but the tippers whose names we display is 2; the index would be 1, and we wouldn't want to display the comma separator

File: packages/web/src/components/tipping/feed-tip-tile/FeedTipTile.tsx [hunk 3]
```diff
@@ -36,32 +58,51 @@ const WasTippedBy = () => (
...
+          index < NUM_FEED_TIPPERS_DISPLAYED - 1 ? (
```
[the remark this answers is not in the document]
Comment (sddioulde) [reply packages/web/src/components/tipping/feed-tip-tile/FeedTipTile.tsx:90]: yup, can be false. basically for the last tipper. similar to above

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-tip-qa

**Pull request merged**